### ISSUE TITLE
Missing Tokenizer Files

### ISSUE THREAD

**Issue by karimfayed** [google-research/pegasus#189]
Hi,
I have used [this script](https://gist.github.com/jiahao87/50cec29725824da7ff6dd9314b53c4b3) to fine-tune PEGASUS and it produced the following files at each checkpoint:
- config.json
- pytorch_model.bin
- optimizer.pt
-  scheduler.pt
- training_args.bin
- trainer_state.json
I uploaded all these files **_except optimizer.pt_** on a new model on huggingface, but unfortunately whenever I use it using [huggingface usage script](https://huggingface.co/transformers/model_doc/pegasus.html#usage-example) , the following error shows up:
![Screenshot (40)](https://user-images.githubusercontent.com/46823709/117587632-3ec2fa00-b11f-11eb-8cf3-173ad751292c.png)

**Notes:**

**1. I don't know should I create my own tokenizer files and add them to the model or should these files be produced with the checkpoint files?**

**2. I replicated tokenizer files of other models and tried to add them to the model ([my model](https://huggingface.co/Karimfayed/pegasus_SAMSum/tree/main)) to see if it would work. It produced the following output along with a totally unexpected output as a  summerization for the source text**

> Some weights of the model checkpoint at Karimfayed/pegasus_SAMSum were not used when initializing PegasusForConditionalGeneration: ['lm_head.weight']
> - This IS expected if you are initializing PegasusForConditionalGeneration from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
> - This IS NOT expected if you are initializing PegasusForConditionalGeneration from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
